scripts/ci/vagrant.sh

```diff
@@ -57,6 +57,11 @@ fedora-no-vdso() {
 }
 
 fedora-rawhide() {
+	# The 6.2 kernel of Fedora 38 in combination with rawhide userspace breaks
+	# zdtm/static/socket-tcp-nfconntrack. To activate the new kernel previously
+	# installed this reboots the VM.
+	vagrant reload
+	ssh default uname -a
```

Comment:
Would it be better to move `vagrant reload` in `setup()` for consistency across tests?

Reply:
I was thinking about it, but if we do not need the latest kernel rebooting takes additional time.